worker/src/RTC/SctpAssociation.cpp

```diff
@@ -356,15 +356,6 @@ namespace RTC
 
 		// Add maxMessageSize.
 		jsonObject["maxMessageSize"] = this->maxSctpMessageSize;
-
```

Comment:
Why this removals?

Reply:
Answered in reply to corresponding test change above.